particles/TestingExample/UnderTest.arcs

```diff
@@ -0,0 +1,6 @@
+meta
+  namespace: arcs.unit.test.example
+
```

Comment:
I don't think particles/ gets imported. Could you move this to somewhere under javatests/ instead? Maybe javatests/arcs/sdk/testing to live alongside the testing code?

Reply:
Given you are arguing for having an example of how to do something inside the java / javatests tree, I've created `arcs.sdk.examples` package for canonical examples of how to use the SDK and a `unit_testing` directory under it for this example in particular.